src/main/java/dk/cngroup/lentils/entity/Cypher.java

```diff
@@ -37,7 +37,7 @@
     private Point location;
 
     @Column(name = "place_description", length = 1000)
-    @Length(max = 1000, message = "Popis místa nesmí být delší než 1000 znaků.")
```

Comment:
Tohle teď akorát bude potřeba přidat do CypherFormObject objektu. Teďkom se ve formuláři používá tento objekt a validují se vstupy na něm.

Reply:
Plus změnit i to číslo v té chybové hlášce :)